test/api/test_configuration.py

```diff
@@ -34,6 +41,22 @@ def test_admin_user_configuration(self):
         assert_has_keys(config, *TEST_KEYS_FOR_ALL_USERS)
         assert_has_keys(config, *TEST_KEYS_FOR_ADMIN_ONLY)
 
+    def test_admin_decode_id(self):
+        new_lib = self.library_populator.new_library('DecodeTestLibrary')
+        decode_response = self._get("configuration/decode/" + new_lib["id"], admin=True)
+        response_id = decode_response.json()["decoded_id"]
+        decoded_library_id = self.security.decode_id(new_lib["id"])
+        assert decoded_library_id == response_id
+        # fake valid folder id by prepending F
+        valid_encoded_folder_id = 'F' + new_lib["id"]
+        folder_decode_response = self._get("configuration/decode/" + valid_encoded_folder_id, admin=True)
+        folder_response_id = folder_decode_response.json()["decoded_id"]
+        assert decoded_library_id == folder_response_id
+
+    def test_normal_user_decode_id(self):
+        decode_response = self._get("configuration/decode/badhombre", admin=False)
```

Comment:
(I trust the `@require_admin` flag, but) shouldn't we test decoding a valid ID with admin=False?

Reply:
this technically tests that, since the response won't be 403